src/main/java/com/lambdatest/jenkins/freestyle/report/AppAutomationReportBuildAction.java

```diff
@@ -62,7 +62,7 @@ public void generateLambdaTestAppAutomationReport() {
                     if (buildDetailNode.get("name").toString().replaceAll("\"", "").equals(buildName)) {
                         String build_id = buildDetailNode.get("build_id").toString();
 
-                        lambdaTestBuildDeviceUrl = Constant.APP_AUTOMATION_APP_URL + "/test?buildid=" + build_id;
+                        lambdaTestBuildDeviceUrl = "https://appautomation.lambdatest.com/build/?searchtxt=" + build_id;
```

Comment:
@Shahnawaz-Sk use Host name as a  constant,
for https://appautomation.lambdatest.com